snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: modem-manager
-version: 1.20.0-3-dev
+version: 1.23.4-0ubuntu2
```

Comment:
It would be 1.23.4-1-dev, we do not follow the same version as the Ubuntu package as there are other things that can change so we might need to bump the version without changing the mm source package.

Reply:
Right - I see, that makes sense